src/pages/components/tooltip-modal.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Tooltip Modal
+description: >
+  A Tooltip modal is a small overlay that opens automatically or through user-action, and displays contextual information for its paired element.
+package: '@zendeskgarden/react-modals'
+components:
+  - modals/src/elements/TooltipModal/TooltipModal.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To provide actions within a tooltip</li>
+      <li>To provide a focus jail when actions need to be inside a tooltip</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To reveal read-only helpful information about an element on hover, use a
+          [Tooltip](/components/tooltip)
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A Tooltip Modal must contain a header and a way to exit the experience.
+
+import Default from '../../examples/tooltip-modal/TooltipModalDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/tooltip-modal/TooltipModalDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Placement
+
+A Tooltip modal component can contain a call-to-action, an icon, and
+a progress indicator.
+
+There are 13 placement options available. The tooltip modal will occupy
+the top position by default. `auto` will default it to a position with
```

Comment:
This `auto` sentence needs grammatical restructure to begin with a capital.

Reply:
We could update this to be:

"The `auto` option will default it to a position with the most available space."

Reply:
Updated.